compio/Cargo.toml

```diff
@@ -14,31 +14,34 @@ repository = "https://github.com/Berrysoft/compio"
 all-features = true
 default-target = "x86_64-pc-windows-msvc"
 targets = [
-    "x86_64-pc-windows-msvc",
```

Comment:
our formatter seems a little different

Reply:
It's actually I configured to use 2 space to indent instead of 4. No matter what, fixed now.